app/views/widgets/ad_landing_page/_masthead.html

```diff
@@ -25,14 +25,7 @@ <h2 class="subtitle">Earn your Masters Degree at Dodge College of Film and Media
       <h2>Take the first step to advance your career.</h2>
 
       <div class="calls-to-action">
-        <div id="form_a4517720-e7d7-4e0e-bafd-2c1045918e0d">Loading...</div>
-        <script>/*<![CDATA[*/
-          var script = document.createElement('script');
-          script.async = 1;
-          script.src = 'https://go.chapman.edu/register/?id=a4517720-e7d7-4e0e-bafd-2c1045918e0d&output=embed&div=form_a4517720-e7d7-4e0e-bafd-2c1045918e0d' + ((location.search.length > 1) ? '&' + location.search.substring(1) : '');
-          var s = document.getElementsByTagName('script')[0];
-          s.parentNode.insertBefore(script, s);
-        /*]]>*/</script>
+        <div id="form_4c37abdc-d5ba-4313-99b6-89b7a869ac37">Loading...</div><script async="async" src="https://go.chapman.edu/register/?id=4c37abdc-d5ba-4313-99b6-89b7a869ac37&amp;output=embed&amp;div=form_4c37abdc-d5ba-4313-99b6-89b7a869ac37">/**/</script>
```

Comment:
This updated code was provided by Shawn and will eventually change depending on the campaign.